fe/fe-core/src/main/java/com/starrocks/sql/optimizer/task/RewriteTask.java

```diff
@@ -0,0 +1,101 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+package com.starrocks.sql.optimizer.task;
+
+import com.google.common.base.Preconditions;
+import com.starrocks.sql.optimizer.ExpressionContext;
+import com.starrocks.sql.optimizer.OptExpression;
+import com.starrocks.sql.optimizer.operator.pattern.Pattern;
+import com.starrocks.sql.optimizer.rule.Rule;
+
+import java.util.List;
+
+public class RewriteTask extends OptimizerTask {
```

Comment:
Please add comment to describe this task, which can help understand the mechanism of optimization